include/grpcpp/security/tls_certificate_verifier.h

```diff
@@ -116,7 +118,7 @@ class CertificateVerifier {
   // verification request is pending.
   //
   // request: the verification information associated with this request
-  void Cancel(TlsCustomVerificationCheckRequest* request);
+  void Cancel(TlsCustomVerificationCheckRequest* request, const absl::Status&);
```

Comment:
nit: Should we add a variable name for the `absl::Status` parameter, for consistency?

Similarly below.